metpy/calc/basic.py

```diff
@@ -229,6 +229,57 @@ def heat_index(temperature, rh, mask_undefined=True):
     return hi
 
 
+@exporter.export
+@check_units(temperature='[temperature]', speed='[speed]')
+def feels_like(temperature, rh, speed, face_level_winds=False):
+    r"""Calculate the current feels like temperature.
+
+    Calculates the current feels like temperature based on the wind chill or heat index
+    as appropriate for the current conditions.
+
+    Parameters
+    ----------
+    temperature : `pint.Quantity`
+        The air temperature
+    rh : `pint.Quantity`
+        The relative humidity expressed as a unitless ratio in the range [0, 1].
+        Can also pass a percentage if proper units are attached.
+    speed : `pint.Quantity`
+        The wind speed at 10m.  If instead the winds are at face level,
+        `face_level_winds` should be set to `True` and the 1.5 multiplicative
+        correction will be applied automatically.
+    face_level_winds : bool, optional
+        A flag indicating whether the wind speeds were measured at facial
+        level instead of 10m, thus requiring a correction.  Defaults to
+        `False`.
+
+    Returns
+    -------
+    `pint.Quantity`
+        The corresponding feels like value(s)
+
+    See Also
+    --------
+    heat_index, windchill
+
+    """
+    wind_chill_temperature = windchill(temperature, speed, face_level_winds=face_level_winds,
+                                       mask_undefined=True)
+
+    heat_index_temperature = heat_index(temperature, rh, mask_undefined=True)
+
+    # Combine the heat index and wind chill arrays (no point has a value in both)
+    feels_like = np.ma.array(wind_chill_temperature.filled(1) *
+                             heat_index_temperature.filled(1),
+                             mask=(wind_chill_temperature.mask * heat_index_temperature.mask))
```

Comment:
For the masks, how about `&` instead of multiply? That would seem more obvious to me.